internal/repository/article.go

```diff
@@ -0,0 +1,11 @@
+package repository
+
+import (
+	"context"
+
+	"github.com/PichuChen/go-bbs"
+)
+
+func (repo *repository) GetPopularArticles(ctx context.Context) ([]bbs.ArticleRecord, error) {
+	return []bbs.ArticleRecord{}, nil
```

Comment:
```suggestion
        // TODO: should implement popular articles there 
	return []bbs.ArticleRecord{}, nil
```

這邊的程式碼看起來是 Mocking 應該要加上 TODO 讓其他人知道這邊不應該固定回傳空 Slice 。

Reply:
fixed, thanks~